src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import classnames from 'classnames';
+
+interface Props {
+  onStatusChanged: (newStatus: string) => void
```

Comment:
Consider using 'TodoStatus' enum instead of string for the 'newStatus' parameter type.